plugins/json/tests/json_02_multi_datatypes_expected.json

```diff
@@ -0,0 +1,28 @@
+[
+{
+    "var_string": [
+        "string12"
+    ]
+},
+{
+    "var_array": [
+        11,
+        12,
+        13
+    ]
+}
+,
+{
+    "var_record": {
+        "var_array": [
+            11,
+            12,
+            13
+        ],
```

Comment:
Why you have "var_array" in the "var_record"?

Reply:
because var_array is a member in the record